models/data_warehouse.model.lkml

```diff
@@ -3899,6 +3912,9 @@ explore: performance_events {
   label: " Performance Events"
   group_label: " Product: Messaging"
   hidden: no
+  sql_always_where: ${duration} < (SELECT PERCENTILE_CONT(.95) WITHIN GROUP (ORDER BY DURATION)
```

Comment:
Oh sorry @sri-byte, I think I misunderstood your DM about it applying to the whole table. What I'm really looking for is for "average of the middle 90%" to just be another measure so that we can have charts with that. The other measures should still be across the full range and not be the middle 90%. Not sure if that's possible or not.

Reply:
I see, okay. Let me take this back, and re-think how to approach this.